### DISCUSSION TITLE
Error while packages are installed and enabled.

### DISCUSSION THREAD

**Discussion by baulaa** [Security-Onion-Solutions/securityonion#3814] [General]
After ubuntu install with required partition sizes, then use the following command

git clone https://github.com/Security-Onion-Solutions/securityonion
cd securityonion
**sudo bash so-setup-network**

Please suggest. 

![image](https://user-images.githubusercontent.com/66031860/114156476-25f1d980-9912-11eb-88a3-348140819a60.png)

**Comment by baulaa** [comment]
unable to install SOC in ubuntu 18.04, however change the distro to centos, now its working fine.

**Reply by dbui1910** [reply]
I'm having the same issues. Did you fix this already?

**Reply by baulaa** [reply]
Never make to work with ubuntu however installed on centos then its working fine? suggest to change the distro. running okay right now.
